screens/shoppingList

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image } from 'react-native';
+import FontLoad from './components/FontLoad';
+
+export default class ShoppingList extends React.Component {
+
+
+  render() {
+    return (
+      this.state.fontLoaded ? (
```

Comment:
This line doesn't make sense - there's no such dictionary variable `this.state` with the key value `fontLoaded`

Also, you don't need to check to see if the font is loaded in this file - `App.js` already makes the one time check

Remove this line